src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -723,4 +723,9 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Property SourceProvider is null.</value>
     <comment>Don't localize 'SourceProvider'</comment>
   </data>
+  <data name="Error_AttemptingToPackSDKproject" xml:space="preserve">
+    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` to pack this project instead. You may revert to the legacy pack behavior by setting the  '{0}' environment variable to 'true'.</value>
```

Comment:
```suggestion
    <value>The `pack` command with nuget.exe for SDK-style projects is not supported, use `dotnet pack` or `msbuild /t:pack` to pack this project instead. To force nuget.exe to run the pack command, you can set the '{0}' environment variable to 'true' but this is not recommended.</value>
```

Reply:
The first part of the message was something we reviewed during the design stage. https://github.com/NuGet/Home/pull/12088#pullrequestreview-1104954849

We didn't call out nuget.exe as it was considered redundant. 

I like your version of the 2nd sentence a lot better than what I had.

Reply:
```suggestion
    <value>The `nuget pack` command for SDK-style projects is not supported, use `dotnet pack` or `msbuild -t:Pack` to pack this project instead. To force the `nuget pack` command through, set the '{0}' environment variable to 'true' but this is neither supported nor recommended.</value>
```

How about this?

Reply:
I kind of combined both of your proposals + the feedback in previous iterations and landed on: 

> The `pack` command for SDK-style projects is not supported, use `dotnet pack` to pack this project instead. To force nuget to run `pack` command, you can set the '{0}' environment variable to 'true' but this is not recommended.

Reply:
I think the "this is not recommended" is redundant. We already provide the suggested alternative.

Here's my modified proposal from above:

> The `pack` command for SDK-style projects is not supported, use `dotnet pack` or `msbuild /t:pack` to pack this project instead. You can override this behavior by setting the '{0}' environment variable to 'true'.